python/run-tests.py

```diff
@@ -160,11 +159,13 @@ def run_individual_python_test(target_dir, test_name, pyspark_python):
 
 
 def get_default_python_executables():
-    python_execs = [x for x in ["python2.7", "python3.6", "pypy"] if which(x)]
-    if "python2.7" not in python_execs:
-        LOGGER.warning("Not testing against `python2.7` because it could not be found; falling"
-                       " back to `python` instead")
-        python_execs.insert(0, "python")
+    python_execs = [x for x in ["python3.6"] if which(x)]
+
+    if "python3.6" not in python_execs:
+        p = which("python3")
```

Comment:
do you need to insert into `python_execs`?

Reply:
good catch.  yes.